### PR TITLE
Pack200 remove from nb installer

### PR DESCRIPTION
**Pull request by singh-akhilesh**
https://issues.apache.org/jira/browse/NETBEANS-3985

### CODE REVIEW

**Comment by graben** [comment]
@singh-akhilesh:
What's the benefit of commenting rather than deleting the code? Nobody will change it and history is still available.

**Comment by rcpacheco** [comment]
I also think that it is better to eliminate the old code, than to leave the code commented. I think that the proyect has a coding style guideline, but don't know where it is. Also, there are other modules that use Pack200, like netbeans-mavenutils-nbm-maven-plugin , see org.apache.netbeans.nbm.CreateClusterAppMojo

**Comment by rcpacheco** [comment]
There seems to be an error with the creation of the Visual Studio Code extension: [exec] [ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:java (default-cli) on project prepare-bundles: An exception occured while executing the Java class. Cannot find license for: agent-base

File: nbbuild/antsrc/org/netbeans/nbbuild/MakeNBM.java [hunk 1]
```diff
@@ -334,14 +334,14 @@ public void setProductDir( File dir ) {
     public void setFile(File file) {
         this.file = file;
     }
-
-    public void setUsePack200(boolean usePack200) {
+    //Commented below code, as pack200 is not supported from jdk 14 onwards
```
Comment (jlahoda) [inline nbbuild/antsrc/org/netbeans/nbbuild/MakeNBM.java:337]: I guess the methods (and so their attributes in the build.xml) should stay here, for compatibility. They, of course, won't do anything.

Comment (singh-akhilesh) [reply nbbuild/antsrc/org/netbeans/nbbuild/MakeNBM.java:337]: Indeed, i'll address this comment.

**Comment by graben** [comment]
> There seems to be an error with the creation of the Visual Studio Code extension: [exec] [ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:java (default-cli) on project prepare-bundles: An exception occured while executing the Java class. Cannot find license for: agent-base

Have a look on #2105

**Comment by ebarboni** [comment]
> 
> 
> > There seems to be an error with the creation of the Visual Studio Code extension: [exec] [ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:java (default-cli) on project prepare-bundles: An exception occured while executing the Java class. Cannot find license for: agent-base
> 
> Have a look on #2105

#2105 merged now hope it help

File: nbbuild/antsrc/org/netbeans/nbbuild/MakeNBM.java (hunk 1 above)
Comment (singh-akhilesh) [reply nbbuild/antsrc/org/netbeans/nbbuild/MakeNBM.java]: Addressed code review comments and removed all the previously commented code.

**Comment by singh-akhilesh** [comment]
> @singh-akhilesh:
> What's the benefit of commenting rather than deleting the code? Nobody will change it and history is still available.

Agree. Deleted the previously commented code.

**Comment by singh-akhilesh** [comment]
@ebarboni Please consider this PR as a candidate of 12.0 release.

**Comment by ebarboni** [comment]
yep merging on beta 4 we may try the installer (the untrusted one) to check on jdk14

**Comment by arvindaprameya** [comment]
+ 1 Approved, Thanks for completing this Akhilesh

**Pull request merged**